### DISCUSSION TITLE
Change default values in settings.ini

### DISCUSSION THREAD

**Discussion by ozapft** [aldostools/IRISMAN#25] [General]
Hallo Aldo,

what do you think about changing the following default settings in settings.ini?

Change TimeoutByInactivity=1 to 0 
1=quit IRISMAN after 1 hour of inactivity (2=2hours ...)
0=disable inactivity timer 

The default value 1 can confuse some Users. 

Change OptionsShowPIC1=0 to 1 
PIC1 of the game will be shown as background when you press triangle for game options.

Looks nice and works flawless

**Comment by aldostools** [comment]
Done. This was a simple repack of the PKG with the values changed in settings_default.ini
The PKG is updated in the release page.

If you already have IRISMAN installed you will not notice any difference.
To see the changes need to delete IRISMAN and reinstall it or delete the settings.ini then reinstall so it read the default values.

**Comment by ozapft** [comment]
Thank you Aldo!
